<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="drink_button">
         <property name="styleSheet">
          <string notr="true">QRadioButton{
    border-radius:8px;
}
QRadioButton::indicator {
	font: 700 12pt &quot;Microsoft YaHei UI&quot;;
    width: 0px;
    height: 20px;
}

QRadioButton::indicator:disabled{
    background-color: #7f8c8d;
    color: white
}
QRadioButton::checked{
    background-color: #7f8c8d;
    color: white
}
QRadioButton:hover:checked{
    background-color: #7f8c8d;
    color: white
}
QRadioButton:hover{
    background-color: #bdc3c7;
    color: white
}</string>
         </property>
         <property name="text">
          <string>喝水</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="class_button">
         <property name="styleSheet">
          <string notr="true">QRadioButton{
    border-radius:8px;
}
QRadioButton::indicator {
	font: 700 12pt &quot;Microsoft YaHei UI&quot;;
    width: 0px;
    height: 20px;
}

QRadioButton::indicator:disabled{
    background-color: #7f8c8d;
    color: white
}
QRadioButton::checked{
    background-color: #7f8c8d;
    color: white
}
QRadioButton:hover:checked{
    background-color: #7f8c8d;
    color: white
}
QRadioButton:hover{
    background-color: #bdc3c7;
    color: white
}</string>
         </property>
         <property name="text">
          <string>上课</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="setting_button">
         <property name="styleSheet">
          <string notr="true">QRadioButton{
    border-radius:8px;
}
QRadioButton::indicator {
	font: 700 12pt &quot;Microsoft YaHei UI&quot;;
    width: 0px;
    height: 20px;
}

QRadioButton::indicator:disabled{
    background-color: #7f8c8d;
    color: white
}
QRadioButton::checked{
    background-color: #7f8c8d;
    color: white
}
QRadioButton:hover:checked{
    background-color: #7f8c8d;
    color: white
}
QRadioButton:hover{
    background-color: #bdc3c7;
    color: white
}</string>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="drink_page">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>喝水提醒间隔时间(秒)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="drink_interval"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="drink_on">
             <property name="text">
              <string>开启 </string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="drink_words">
             <property name="text">
              <string>词库</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="setting_page">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="3">
            <widget class="QPushButton" name="submit_title">
             <property name="text">
              <string>确认修改</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>修改标题名称</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="4">
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="if_start_on_boot">
             <property name="text">
              <string>开机自启</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="class_page">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="6">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="class_on_notify">
             <property name="text">
              <string>上课提醒 </string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>提前提醒时间(分) </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QPushButton" name="import_class">
             <property name="text">
              <string>导入课程</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3" colspan="2">
            <widget class="QPushButton" name="add_class">
             <property name="text">
              <string>添加课程</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QCheckBox" name="class_off_notify">
             <property name="text">
              <string>下课提醒 </string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="5">
            <widget class="QLineEdit" name="class_notify_before"/>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
